dev/source/docs/sitl-with-morse.rst

```diff
@@ -3,6 +3,9 @@
 =====================
 Using SITL with Morse
 =====================
+.. warning::
+
+    Morse robotics simulator is no longer supported!
```

Comment:
Maintained, we still support its backend on ArduPilot side

Reply:
Thanks for this @khancyr  sir.Got a little confused while reading the Morse Github. Nevertheless have made the changes